WireMessaging/Package.swift

```diff
@@ -64,26 +51,17 @@ let package = Package(
             plugins: [.plugin(name: "SwiftGenPlugin", package: "WirePlugins")]
         ),
         .target(
-            name: "WireMessagingImplementationSupport",
+            name: "WireMessagingDomainSupport",
```

Comment:
question: seeing as WireMessageDomain is a product, I can understand the need for this support target so other modules can access mocks for public domain protocols. But I imagine that ideally the domain target is't publicly accessible and there would be no need for protocol mocks to be accessible outside the package. What do you think?

Reply:
I think though the test targets could also depend on domain and the mocks are available in debug builds... not sure what's better.